x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/_dimension_popover.scss

```
@@ -1,6 +1,7 @@
 .lnsDimensionPopover {
   line-height: 0;
   flex-grow: 1;
+  max-width: calc(100% - 32px);
```

Need to use eui vars for all px values
```suggestion
  max-width: calc(100% - ${euiSizeXL});
```